packages/plugins/src/hardhat/tasks.ts

```diff
@@ -1375,3 +1378,182 @@ export const listProjectsTask = async ({}, hre: HardhatRuntimeEnvironment) => {
 task(TASK_CHUGSPLASH_LIST_PROJECTS)
   .setDescription('Lists all projects that are owned by the caller.')
   .setAction(listProjectsTask)
+
+export const claimProxyTask = async (
+  args: {
+    configPath: string
+    referenceName: string
+    silent: boolean
+  },
+  hre: HardhatRuntimeEnvironment
+) => {
+  const { configPath, referenceName, silent } = args
+  const provider = hre.ethers.provider
+  const signer = provider.getSigner()
+
+  const spinner = ora({ isSilent: silent })
+  spinner.start('Checking project registration...')
+
+  const parsedConfig = loadParsedChugSplashConfig(configPath)
+
+  // Throw an error if the project has not been registered
+  if (
+    (await isProjectRegistered(signer, parsedConfig.options.projectName)) ===
+    false
+  ) {
+    errorProjectNotRegistered(
+      await getChainId(hre.ethers.provider),
+      hre.network.name,
+      configPath
+    )
+  }
+
+  const owner = await getProjectOwnerAddress(
+    provider,
+    parsedConfig.options.projectName
+  )
+
+  const signerAddress = await signer.getAddress()
+  if (owner !== signerAddress) {
+    throw new Error(
+      `Caller does not own the project ${parsedConfig.options.projectName}`
+    )
+  }
+
+  spinner.succeed('Project registration detected')
+  spinner.start('Claiming proxy ownership...')
+
+  const manager = getChugSplashManager(signer, parsedConfig.options.projectName)
+
+  const activeBundleId = await manager.activeBundleId()
+  if (activeBundleId !== ethers.constants.HashZero) {
+    throw new Error(
+      `A project is currently being executed. Proxy ownership has not been transferred.
+Please wait a couple of minutes before trying again.`
+    )
+  }
+
+  await (
+    await manager.transferProxyOwnership(referenceName, signerAddress)
+  ).wait()
+
+  spinner.succeed(`Proxy ownership claimed by address ${signerAddress}`)
+}
+
+task(TASK_CHUGSPLASH_CLAIM_PROXY)
+  .addPositionalParam(
+    'configPath',
+    'Path to the ChugSplash config file for the project that owns the target contract'
+  )
+  .addPositionalParam(
```

Comment:
Was just messing around with these tasks and realized that having two positional params is pretty unintuitive. E.g. running `npx hardhat chugsplash-claim-proxy <configPath> <referenceName>` works, but switching their order (i.e. `<configPath> <referenceName>`) doesn't. In all of the Hardhat tasks I've used, there's only one positional param to avoid this. What do you think about changing the reference name to `addFlag`?